examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -619,8 +621,8 @@ bool loadConfigFile(void) {
     } else {
       debug("Config file doesn't exist!");
     }
-    debug("Unmounting SPIFFS.");
-    SPIFFS.end();
+    debug("Unmounting FILESYSTEM.");
```

Comment:
`    debug("Unmounting " FILESYSTEM ".");`
Use that instead, and it should report if it is SPIFFS or LittleFS etc.